project.clj

```diff
@@ -24,7 +24,8 @@
              :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]
                                   [clojure-future-spec "1.9.0"]]}
              :1.9 {:dependencies [[org.clojure/clojure "1.9.0"]]}
-             :1.10 {:dependencies [[org.clojure/clojure "1.10.3"]]}}
+             :1.10 {:dependencies [[org.clojure/clojure "1.10.3"]]}
+             :1.11 {:dependencies [[org.clojure/clojure "1.11.1"]]}}
```

Comment:
Please add `1.11` to also `.github/workflows/build.yml`.

https://github.com/chrovis/varity/blob/f84c720b6f4b111d7d97127c5749ff57adea7bec/.github/workflows/build.yml#L11